universities/university-of-vermont.md

```diff
@@ -1 +1,31 @@
+---
+title: "University of Vermont"
```

Comment:
We already have this in a PR. Can you check that version against this and see what you could've done differently?